torch/utils/cpp_extension.py

```diff
@@ -142,15 +144,17 @@ def _find_rocm_home() -> Optional[str]:
     return rocm_home
 
 def _find_sycl_home() -> Optional[str]:
-    """Find the OneAPI install path."""
-    # Guess #1
-    sycl_home = os.environ.get('ONEAPI_ROOT')
-    if sycl_home is None:
-        # Guess #2
-        icpx_path = shutil.which('icpx')
-        if icpx_path is not None:
-            sycl_home = os.path.dirname(os.path.dirname(
-                os.path.realpath(icpx_path)))
+    sycl_home = None
+    icpx_path = shutil.which('icpx')
```

Comment:
Could you help add comments to elaborate on the underlying logic?

Reply:
Resolved， thanks.